templates/contact.html

```diff
@@ -10,10 +10,11 @@ <h3 class="text-secondary display-5 mb-4"><i class="text-primary pr-2 fas fa-tab
       <p class="text-tpo"><span class="text-primary nick">#tor-dev</span> - {{ _("Discuss Tor-related coding and protocols. Ideas are welcome.") }}</p>
       <p class="text-tpo"><span class="text-primary nick">#tor-l10n</span> - {{ _("Get in touch with other translators") }}</p>
       <p class="text-tpo"><span class="text-primary nick">#tor-meeting</span> - {{ _("Watch or join publicly logged team meetings.") }}</p>
-      <p class="text-tpo"><span class="text-primary nick">#tor-project</span> - {{ _("Discuss organization and community related topics: meetups, outreach, translation, or website improvements.") }}</p>
+      <p class="text-tpo"><span class="text-primary nick">#tor-project</span> - {{ _("Discuss organization and community related topics: meetups, outreach, and translation.") }}</p>
```

Comment:
Maybe we could also take translations out of this line, as now there is #tor-l10n too.